ui/components/component-library/checkbox/README.mdx

```diff
@@ -12,7 +12,7 @@ Checkbox is a graphical element that allows users to select one or more options
 
 ## Props
 
-The `Checkbox` accepts all props below as well as all [Box](/docs/components-ui-box--default-story#props) component props
+The `Checkbox` accepts all props below as well as all [Box](/docs/components-componentlibrary-box--docs#props) component props
```

Comment:
Link to new TS Box component I've created an issue to fix all the links https://github.com/MetaMask/metamask-extension/issues/20004